slack-base/src/main/java/com/hubspot/slack/client/models/users/UserProfileIF.java

```diff
@@ -28,6 +29,9 @@
   Optional<String> getPhone();
   Optional<String> getSkype();
 
+  // Extra custom fields set by your workspace admin
+  Optional<Map<String, ProfileField>> getFields();
```

Comment:
Would be awesome if we could re-use [this deserialization test](https://github.com/HubSpot/slack-client/blob/master/slack-base/src/test/java/com/hubspot/slack/client/models/users/SlackUserDeserializationTest.java) to verify these fields are parsed